programs/ssl/ssl_test_lib.h

```diff
@@ -219,5 +219,36 @@ int idle( mbedtls_net_context *fd,
 #endif
           int idle_reason );
 
+#if defined(MBEDTLS_TEST_HOOKS)
+/** Initialize whatever test hooks are enabled by the compile-time
+ * configuration and make sense for the TLS test programs. */
+void test_hooks_init( void );
+
+/** Check if any test hooks detected a problem.
+ *
+ * If a problem was detected, make sure to print an explanation to stderr,
+ * either at the time of detection or during the call to this function.
```

Comment:
I find this unclear. Is this a description of what this function does, of what the caller should do, or what somebody else should does or should do?